src/assets/stylesheets/global.scss

```diff
@@ -1,8 +1,8 @@
+@import 'normalize';
```

Comment:
so right now, this normalize (and any 'global' selectors) is a major issue. every time we import global into another scss file, it brings in ALL of these classes and assigns them module names

this means, if we have 5 files importing `global`, then all of the `global` selectors will be included in the final css 5 times, causing huge bloat

we can discuss as a team solution